ci/cloudbuild/notifiers/bigquery/README.md

```diff
@@ -0,0 +1,20 @@
+This directory contains files needed to use the Cloud Build BigQuery notifier.
```

Comment:
nit: a title? `# Configuration files for Cloud Build BigQuery notifier`

And then explain what is the problem solved by this notifier thingy?

```
Sometimes we need to analyze our build results, looking for patterns of build latency, flakiness, etc. These analysis often require looking at the results over long periods of time, so the usual dashboards and squinting may not work. Putting the results in some sort of database works better for that purpose.  The Cloud Build BigQuery notifier allows us to store build results in BigQuery.
```

Reply:
Thanks, done.